src/main/java/fruitshop/service/impl/DataConverterImpl.java

```diff
@@ -0,0 +1,27 @@
+package fruitshop.service.impl;
+
+import fruitshop.model.FruitTransaction;
+import fruitshop.service.DataConverter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    public static final int TYPE = 0;
+    public static final int FRUIT = 1;
+    public static final int QUANTITY = 2;
+    public static final String COMA = ",";
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> inputReport) {
+        List<FruitTransaction> result = new ArrayList<>();
+        for (String fruitReport : inputReport) {
+            String[] splitInputReport = fruitReport.split(COMA);
```

Comment:
It's recommended to handle the case where the input line does not contain the expected number of elements after splitting. This can prevent `ArrayIndexOutOfBoundsException` if the data is malformed.